distribution/docs/src/main/resources/_contents/_catalog-frameworks/attribute-type-definition-contents.adoc

```diff
@@ -0,0 +1,68 @@
+
+==== Attribute Type Definition File
+
+To define Attribute Types, the definition file must have an `attributeTypes` key in the root object.
+
+[source,json]
+----
+{
+    "attributeTypes": {...}
+}
+----
+
+The value of `attributeTypes` must be a map where each key is the attribute type's name and each value is a map that includes the data type and whether the attribute type is stored, indexed, tokenized, or multi-valued.
```

Comment:
I think @pklinef will be able to give you the best definitions but here are some starter ones:

- `stored`: If true, the value of the attribute should be stored in the underlying datastore. Some attributes may only be indexed or used in transit and do not need to be persisted.
- `indexed`: If true, then the value of the attribute should be included in the datastore's index and therefore be part of query evaluation.
- `tokenized`: Only applicable to `STRING_TYPE` attributes, if true then [stopwords](https://en.wikipedia.org/wiki/Stop_words) will be stripped prior to storing and/or indexing.
- `multi-valued`: If true, then the attribute values will be Lists of the attribute type rather than single values.